test/cancel.ts

```diff
@@ -157,30 +157,38 @@ test('recover from cancellation using error instance', async t => {
 });
 
 test('throws on incomplete (canceled) response - promise', withServer, async (t, server, got) => {
-	server.get('/', downloadHandler);
+	server.get('/', downloadHandler(got));
 
 	await t.throwsAsync(got({
-		timeout: {request: 500}
-	}), got.TimeoutError);
+		timeout: {request: 500},
+		retry: 0
+	}).then(() => null), got.TimeoutError);
```

Comment:
Use async/await, not `.then`.

Reply:
`async/await` does not apply here... But I have removed this change anyways.